documentation/sandbox-guide/movies.adoc

```diff
@@ -0,0 +1,315 @@
+﻿// https://docs.google.com/a/neotechnology.com/presentation/d/1uajyGl64zdpHjD1d92hQVEJErdnnZ9bACJF7J-TnEWI/edit?usp=drive_web
+// https://www.vmware.com/pdf/vi_architecture_wp.pdf
+// ping Alan, share document
+++++
+<style type="text/css">
+.smallest {
+   font-size:0.6em;
+}
+</style>
+++++
+
+:neo4j-version: 4.x
+:author: Shreyans Gandhi
+:tags: movies
+:images: https://dl.dropboxusercontent.com/u/14493611
+:images: {img}
+:experimental:
+
+==  What is Cypher?
+
+Cypher is a graph query language that is used to query the Neo4j Database. Just like you use SQL to query a MySQL database, 
+you would use Cypher to query the Neo4j Database.
+
+A simple cypher query can look something like this 
+
+[source,cypher]
+----
+Match (m:Movie) where m.released > 2000 RETURN m limit 5
+----
+
+++++
+<div style="background: #5CA6D9EE; padding: 12px; color: white; border-radius: 2px">

```

Comment:
cannot standard [admonition blocks](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/) be used here?

Reply:
The standard block design won't fit well with the browser color scheme. So let's keep this as is for now. We are anyways going to work to standardise guides to a newer format  (most likely MDX).